lua/starfall/libs_cl/bass.lua

```diff
@@ -87,7 +87,7 @@ end
 -- @param flags that will control the sound
 -- @param callback to run when the sound is loaded
 function bass_library.loadURL (path, flags, callback)
-	SF.Permissions.check(SF.instance.player, nil, "bass.loadURL")
+	SF.Permissions.check(SF.instance, nil, "bass.loadURL")
```

Comment:
You could expose the path to the permission system here

Reply:
Yes someone can do that later if they want to use it for something